### ISSUE TITLE
Adjusting gamma and contrast

### ISSUE THREAD

**Issue by mlibre** [Fushko/gammy#94]
Adjusting gamma and contrast as well would be nice

**Comment by Fushko** [comment]
Yes, this would be really needed, as the app overrides other software's gamma adjustments. There's other priorities right now, but I'll definitely look into it.

**Comment by mlibre** [comment]
Yeh, it overrides.  
KK. Great
